deploy/example_catalog/cr-app-datafabric610.json

```diff
@@ -0,0 +1,232 @@
+{
+  "apiVersion": "kubedirector.hpe.com/v1beta1",
+  "kind": "KubeDirectorApp",
+  "metadata": {
+    "name": "datafabric610"
+  },
+  "spec": {
+    "systemdRequired": true,
+    "defaultPersistDirs": [
+      "/opt/mapr"
+    ],
+    "config": {
+      "configMeta": {
+        "storage": "60",
```

Comment:
what is this storage used for ?

Reply:
fileserver is must for CLDB and nodemanager, in case there are block device attached to the pod, flat-file storage gets created. This storage size is for that flat-file.

Reply:
you mean if there is no block device then it defaults to the previous flat file-based approach?

Reply:
that is correct